src/js/main.js

```diff
@@ -744,14 +743,11 @@ function getConfiguratorVersion() {
 }
 
 function updateTopBarVersion(firmwareVersion, firmwareId, hardwareId) {
```

Comment:
Now this function only updates the "global" variables that contain the versions that are used in the logo components. Maybe better rename to `updateLogoVersions` or something similar? Or do you think is not a good approach reference the component?

Reply:
Happy to change this. But technically this function still does the exact same thing as it used to do 😉 So it's more for a clarity. But keeping it this way means who ever touched the code in the past will roughly know what it's doing.

Reply:
True. But in the original version has code that changed explicit the html with jquery. Now we only change some "variables", maybe renaming the function will give the developer that read the code more information about it.
You don't need to change it now, only was a question/suggestion to see what is your opinion about this or if this is a common practice with Vue/React/etc.